DragaliaAPI.Test/Services/StoryServiceTest.cs

```diff
@@ -195,6 +195,28 @@ public async Task ReadUnitStory_StoryRead_ReturnsExpectedReward(StoryTypes type,
         this.mockStoryRepository.VerifyAll();
     }
 
+    [Theory]
+    [InlineData(StoryTypes.Chara, 100001085, 10150403)]
```

Comment:
Could you add some more cases? Not much point of a `[Theory]` with only one case